### ISSUE TITLE
[Enhancement] Add Kannada language

### ISSUE THREAD

**Issue by schlagmichdoch** [schlagmichdoch/PairDrop#227]
Hi @schlagmichdoch, I would like to translate PairDrop to the Kannada language, can you please add it?

_Originally posted by @ch3thanhs in https://github.com/schlagmichdoch/PairDrop/issues/35#issuecomment-1832352354_

**Comment by schlagmichdoch** [comment]
@ch3thanhs

I have forgotten to enable your translations when I deployed `v1.10.0` some days ago but I have added it today into `v1.10.2` which is running live on https://pairdrop.net/ :tada: 

In the future it's best to create a new issue for such things instead of discussing it in closed issues to have a better overview. I will also break out any issues in that regard.

Thanks for your contribution and have fun with the new version!
